docs/topics/templates/document-attributes.adoc

```diff
@@ -104,16 +119,14 @@ endif::[]
 :LinkAPI: http://windup.github.io/windup/docs/latest/javadoc/
 
 //What about these ones?
-//Links to Jira WINDUP and WINUPRULES project pages:
-:JiraWindupURL: https://issues.redhat.com/projects/WINDUP
-:JiraWindupRulesURL: https://issues.redhat.com/projects/WINDUPRULE
-
-// Point to the GitHub wiki version since there is no product version of this guide.
-:ProductDocCoreGuideURL: https://github.com/windup/windup/wiki/Core-Development-Guide
+//Links to MTA and MTR Jira project pages:
+:JiraMTRURL: https://issues.redhat.com/projects/WINDUP
+:JiraMTAURL: https://issues.redhat.com/projects/TACKLE
 
 // URLs for downloads on developers.redhat.com
 :MTADownloadPageURL: https://developers.redhat.com/products/mta/download
```

Comment:
Do {MTADownloadPageURL} and {MTRDownloadPageURL} appear in the docs-- maybe sasha added them, i didn't -- if not, once again in the ifdefs.